src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisonperson/profiledetails/ProfilesDetailsIntTest.kt

```diff
@@ -335,19 +340,21 @@ class ProfilesDetailsIntTest : IntegrationTestBase() {
           }
         }
 
-        service.upsertProfileDetails("A1234AA", UpsertProfileDetailsRequest("BUILD", "SMALL"))
-          .also {
-            assertThat(it.bookingId).isEqualTo(booking.bookingId)
-            assertThat(it.created).isTrue()
-          }
+        repository.runInTransaction {
```

Comment:
Needed because I'm calling the service directly - will go when we call the endpoint instead.